.circleci/config.yml

```diff
@@ -146,6 +146,10 @@ jobs:
 
             # MNIST tests
 
+            # 0) mnist_patch.py
```

Comment:
@trsvchn I think we were too fast with circle ci. We have to install to there (linux machine)
```
FileNotFoundError: [Errno 2] No such file or directory: 'wget'
```
https://app.circleci.com/pipelines/github/pytorch/ignite/1483/workflows/8da2c054-e304-44de-9cf2-6d19ec0853d4/jobs/4243


Maybe, `apt-get update && apt-get install -y wget` can help there. Let me do that from my side.

Reply:
I see, yes, let's try that!

Reply:
https://github.com/pytorch/ignite/pull/1739